### DISCUSSION TITLE
tonic-web: support for Chrome Extensions to debug in network panel

### DISCUSSION THREAD

**Discussion by k-bx** [hyperium/tonic#1451] [Q&A]
I'm not seeing any of the extensions working with my gRPC-Web prototype:

- https://chrome.google.com/webstore/detail/grpc-web-developer-tools/kanmilmfkjnoladbbamlclhccicldjaj
- https://chrome.google.com/webstore/detail/grpc-devtools/fohdnlaeecihjiendkfhifhlgldpeopm

One of them mentions https://grpc.io/blog/grpc-web-interceptor/ requirements.

Is this something supported by tonic-web?

**Comment by kyle-mccarthy** [comment]
@k-bx this has been working for me https://github.com/SafetyCulture/grpc-web-devtools

One thing to note is that it will only capture requests if the dev tools is open to the "gRPC-Web" tab.
